examples/image_classification_onnx/camera.html

```diff
@@ -97,7 +104,7 @@
       </div>
     </div>
     <canvas id="canvas" width=224 height=224></canvas>
-    <script src="../third_party/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```

Comment:
How about using the jquery file in ./third_party?

Reply:
OK. I will change it soon.